pkg/api/core/v1/models/models.go

```diff
@@ -306,6 +306,7 @@ type InfoResponse struct {
 	KubeVersion         string `json:"kube_version,omitempty"`
 	Platform            string `json:"platform,omitempty"`
 	DefaultBuilderImage string `json:"default_builder_image,omitempty"`
+	DEXAvailable        bool   `json:"dex_available,omitempty"`
```

Comment:
I would not expose the technical details, but just that the OIDC provider is available/enabled.

```suggestion
	OIDCEnabled        bool   `json:"oidc_enabled,omitempty"`
```

Reply:
sensible. will do

Reply:
done. commit b26a271778e8f27050ffb3d1fcbedeafd3249806

Reply:
Dex is still exposed via the API field.

Reply:
Ah, swagger. Fixed. 4aba21e4193fa5171c28c84b924ac1ffd239315b

Reply:
It wasn't related only to the swagger but also to the exposed JSON in the API:

```go
`json:"dex_available,omitempty"`
```
vs
```go
`json:"oidc_enabled,omitempty"`
```

Reply:
:facepalm: Now truly fixed with commit 6880b87d5a2f0c485067d1149f0237ce9a414d6f